src/main/java/org/isf/distype/manager/DiseaseTypeBrowserManager.java

```diff
@@ -137,4 +137,5 @@ protected void validateDiseaseType(DiseaseType diseaseType, boolean insert) thro
 			throw new OHDataValidationException(errors);
 		}
 	}
+
```

Comment:
this is the only change to this class?

Reply:
Looks like it :)   I did some formatting of files and this happened.